bsp/src/org/jetbrains/bsp/BSPCli.scala

```diff
@@ -59,6 +65,16 @@ object BSPCli extends App {
   val opts = try {
     val opts = parseOpts(args)
     opts.tracePath.fold({})(p => sys.props += ("BSP_TRACE_PATH" -> p))
+    val application = new MockApplication(() => {}) {
+      override def getComponent[T](interfaceClass: Class[T]): T = interfaceClass match {
+        case q if q == classOf[VirtualFileManager] =>
```

Comment:
this looks more like a simple if/else than match

Reply:
Fixed this, also added headless awt property (not present here)